workflow/envs/rbt.yaml

```
@@ -2,4 +2,4 @@ channels:
   - bioconda
   - conda-forge
 dependencies:
-  - rust-bio-tools =0.38
+  - rust-bio-tools =0.38.3
```

```suggestion
  - rust-bio-tools =0.39
```

I think in general, we should not pin the patch number, as it contains backwards compatible bug fixes. If someone installs the workflow, I think we want to make sure, that the latest bug fixes are included. If we need to update an environment, we should delete it, from my point of view :)

However, 0.39 is the newest version so it should also update the envs, as soon as the new master is used.